controller/controller_server/utils.py

```diff
@@ -255,10 +255,13 @@ def get_object_id_info(full_object_id, object_type):
 def get_node_id_info(node_id):
     logger.debug("getting node info for node id : {0}".format(node_id))
     split_node = node_id.split(config.PARAMETERS_NODE_ID_DELIMITER)
-    if len(split_node) != config.SUPPORTED_CONNECTIVITY_TYPES + 1:  # the 1 is for the hostname
+    hostname, iscsi_iqn, fc_wwns = "", "", ""
```

Comment:
I would change to order to `hostname, fc_wwns, iscsi_iqn`

Reply:
done